MuxExoPlayer/src/r2_7_3/java/com/mux/stats/sdk/muxstats/MuxStatsExoPlayer.java

```diff
@@ -255,7 +255,6 @@ public void onLoadCanceled(DataSpec dataSpec, int dataType, int trackType, Forma
 
     @Override
     public void onLoadError(DataSpec dataSpec, int dataType, int trackType, Format trackFormat, int trackSelectionReason, Object trackSelectionData, long mediaStartTimeMs, long mediaEndTimeMs, long elapsedRealtimeMs, long loadDurationMs, long bytesLoaded, IOException e, boolean wasCanceled) {
-        internalError(new MuxErrorException(ERROR_IO, "IOException - " + e.getMessage()));
```

Comment:
Should we not do this for all Exo versions, just to be consistent across them all?